src/Mpdf.php

```diff
@@ -9407,7 +9407,7 @@ function Output($name = '', $dest = '')
 						header('Content-Length: ' . strlen($this->buffer));
 					}
 
-					header('Content-disposition: inline; filename="' . $name . '"');
+					header('Content-Disposition: attachment; filename="' . $name . '"; filename*=utf-8\' \'' . rawurlencode($name));
```

Comment:
Could you give an explanation why this should be changed from inline to attachment?